src/openrct2-ui/windows/RideConstruction.cpp

```diff
@@ -1284,9 +1287,11 @@ class RideConstructionWindow final : public Window
                 }
                 else
                 {
-                    uint8_t* brakesSpeedPtr = &_currentBrakeSpeed2;
-                    uint8_t maxBrakesSpeed = 30;
-                    uint8_t brakesSpeed = *brakesSpeedPtr + 2;
+#ifdef UNIFIED_BRAKE_SPEED_ENABLED
+                    // Increase brake speed
+                    uint8_t* brakesSpeedPtr = &_currentBrakeSpeed;
+                    uint8_t maxBrakesSpeed = std::numeric_limits<uint8_t>::max();
```

Comment:
As a pre-requisite task, verify all RTD OperatingSettings.MaxBrakesSpeed and use the value for maxBrakesSpeed